lib/ansible/modules/cloud/azure/azure_rm_acs.py

```diff
@@ -41,7 +41,8 @@
             - Valid azure location. Defaults to location of the resource group.
     orchestration_platform:
         description:
-            - Specifies the Container Orchestration Platform to use. Currently can be either DCOS, Kubernetes or Swarm.
+            - Specifies the Container I(orchestration_platform) to use. Currently can be either C(DCOS), C(Kubernetes) or C(Swarm).
```

Comment:
no I(orchestration_platform)

here talks about property rather than option

Reply:
Thanks yuwzho. Has changed according to comments!